src/py42/clients/cases.py

```diff
@@ -0,0 +1,175 @@
+from enum import Enum
```

Comment:
This import statement is causing the build to fail on python 2.7 - must import full path

Reply:
Oh I see now... This just is not available in python2, but I don't think it is needed anyway.